src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -125,18 +171,21 @@ def create_uworker_main_batch_job(module_name, cf_job, input_download_url):
   project_id = 'google.com:clusterfuzz'
   region = 'us-central1'
   create_request.parent = f'projects/{project_id}/locations/{region}'
-  result = _create_job(create_request)
+  result = _send_create_job_request(create_request)
   logs.log('Created batch job.')
   return result
 
 
-@retry.wrap(retries=3, delay=2, function='google_cloud_utils.batch._create_job')
-def _create_job(create_request):
+@retry.wrap(
+    retries=3,
+    delay=2,
+    function='google_cloud_utils.batch._send_create_job_request')
+def _send_create_job_request(create_request):
   return _batch_client().create_job(create_request)
 
 
 def get_spec(full_module_name, job_name):
```

Comment:
nit: this wasn't introduced in this PR, but this function is a bit confusing. It seems to imply that `full_module_name`, `job_name` form a unique key for a spec, when it's not. I would suggest instead to have this function take in the exact parameters it needs (i.e. platform, and preemptible/non-preemptible). This is also a lot more modular.